src/config.ts

```diff
@@ -90,37 +90,38 @@ const config = {
       certDuration: 31536000,
     },
     networkConfig: {
-      // ForwardProxy
-      forwardHost: '127.0.0.1' as Host,
-      forwardPort: 0 as Port,
-      proxyHost: '0.0.0.0' as Host,
-      proxyPort: 0 as Port,
       // GRPCServer for agent service
       agentHost: '127.0.0.1' as Host,
       agentPort: 0 as Port,
+      ipv6Only: false,
```

Comment:
It's the config for the QUICSocket. So it applies just to agent communication.

Reply:
Yea that's what I'm saying, I think that should be namespaced somehow either by a subgroup within the config, or it should have a prefix to indicate what it's really intended for.

Also you still have the comment mentioning `GRPCServer` on line 93.

```
// GRPCServer for agent service
```

You should do a fuzzy search for any mention of `GRPC` to get rid of those things.

Reply:
If this is under `networkConfig`, it should apply to both client service and agent service, basically the whole shebang, otherwise it needs to be namespaced like the other options. I like the idea of applying it to the whole system. Consider #511 too...? Or is that for phase 2?

If we just need to merge this, then consider namespacing it.

Reply:
Then of course tag this as a task for phase 2.

Reply:
I'll leave this for phase 2. I'll link this comment in the PR.